README.md

```diff
@@ -242,6 +242,32 @@ else
 end
 ```
 
+You can set options for `#with` to schema by overriding `#with_options`.
```

Comment:
Not sure if I can override `private` method in subclasses. Can I? Please, provide specs for both default and overridden case.

Reply:
```
irb(main):039:0> class A
irb(main):040:1>   private def test
irb(main):041:2>     'hello'
irb(main):042:2>   end
irb(main):043:1> end
=> A
irb(main):044:0>
irb(main):045:0* class B < A
irb(main):046:1>
irb(main):047:1*   def show
irb(main):048:2>   test
irb(main):049:2>   end
irb(main):050:1>
irb(main):051:1*   private def test
irb(main):052:2>     puts "#{super} man"
irb(main):053:2>   end
irb(main):054:1> end
=> B
irb(main):055:0> B.new.show
hello man
=> nil
```